core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6348,6 +6409,62 @@ components:
             Caller must have the add_project action for this service perimeter in
             Sam.
       description: ""
+    SpendReport:
+      required:
+        - spendDetails
+        - spendSummary
+      type: object
+      properties:
+        spendDetails:
+          description: aggregation of spend data
+          type: array
+          items:
+            $ref: '#/components/schemas/SpendReportAggregation'
+        spendSummary:
+          $ref: '#/components/schemas/SpendReportForDateRange'
+    SpendReportAggregation:
+      required:
+        - aggregationKey
+        - spendData
+      type: object
+      properties:
+        aggregationKey:
+          type: object
+          description: aggregation key
+          properties:
+            key:
+              type: string
+              description: key
+        spendData:
+          type: array
+          items:
+            $ref: '#/components/schemas/SpendReportForDateRange'
+    SpendReportForDateRange:
+      required:
+        - cost
+        - credits
+        - currency
+        - startTime
+        - endTime
+      type: object
+      properties:
+        cost:
+          type: string
+          description: total cost for date range
+        credits:
+          type: string
+          description: credits
+        currency:
+          type: string
+          description: currency that cost and credits uses
+        startTime:
+          type: string
+          format: timestamp
+          description: timestamp marking the start of reported spend range (YYYY-MM-DDThh:mm:ss.fffZ)
```

Comment:
Will this be GMT?

Reply:
I'm not sure off the top of my head, but I'll find out and be sure to clarify time zone and specific start and end times here and elsewhere in the swagger docs